app/scripts/proactive/view/FileBrowserView.js

```diff
@@ -126,53 +129,45 @@ define(
             }
             $.ajax({
                 url: that.dataspaceRestUrl + encodeURIComponent(pathname),
-                data: { "comp": "list", "includes": this.filterValue },
+                data: { "comp": "list-metadata", "includes": this.filterValue },
                 headers: { "sessionid": localStorage['pa.session'] },
-                async: false,
                 success: function (data){
-                    that.model['files'] = that.getFilesMetadata(data.fileListing);
-                    that.model['directories'] = that.getFilesMetadata(data.directoryListing);
+                    that.model['files'] = that.getFilesMetadata(data.fileListing, data);
+                    that.model['directories'] = that.getFilesMetadata(data.directoryListing, data);
                     that.model['showHidden'] = that.showHidden;
                     that.$el.html(that.template(that.model));
                     if(that.uploadRequest) {
                         that.switchToUploadingState();
                     }
+                    document.getElementById("filter-files-input").value = that.filterValue;
```

Comment:
can you explain what you do here plz?

Reply:
I just moved this old code. I guess it's for search bar.